src/topology/algebra/infinite_sum.lean

```diff
@@ -71,8 +71,13 @@ lemma has_sum.summable (h : has_sum f a) : summable f := ⟨a, h⟩
 lemma has_sum_zero : has_sum (λb, 0 : β → α) 0 :=
 by simp [has_sum, tendsto_const_nhds]
 
+lemma has_sum_empty [is_empty β] : has_sum f 0 :=
+by convert has_sum_zero
+
 lemma summable_zero : summable (λb, 0 : β → α) := has_sum_zero.summable
 
+lemma summable_empty [is_empty β] : summable f := ⟨0, has_sum_empty⟩
```

Comment:
```suggestion
lemma summable_empty [is_empty β] : summable f := has_sum_empty.summable
```
Matches the style of most of the summable proofs

Reply:
OK, I'll make the change.

Actually, I'm unfamiliar with the `.` notation here (and in your suggestion for rewriting the proof of `tsum_empty`). I'll post the question on Zulip in case it's useful for others.